python/llm/src/bigdl/llm/cmd/wrapper.py

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import os
+import subprocess
+
+LIB_DIR = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "libs")
+
+
+def llama(args):
+    if os.name == 'nt':
+        exec_path = os.path.join(LIB_DIR, 'main-llama.exe')
+    else:
+        exec_path = os.path.join(LIB_DIR, 'main-llama')
+    command = [exec_path,
+               '-t', args.threads,
+               '-p', args.prompt,
+               '-n', args.n_predict,
+               '-m', args.model]
+    subprocess.run(command, check=True)
```

Comment:
Why do we run the command in a separate process? Why not simply run the relevant program through a shell script if this is simply a wrapper?

Reply:
The reason why python was chosen to code the wrapper at that time was also to obtain the model family by reading the model file, because python can read the file more easily than the shell.

Reply:
> The reason why python was chosen to code the wrapper at that time was also to obtain the model family by reading the model file, because python can read the file more easily than the shell.

I don't think this is needed at this stage.